### PR TITLE
FINERACT-2035: Fixed-Client Trend For Day Scale is not work Due to BadSqlGrammar Error

### PR DESCRIPTION
**Pull request by Mariappan-Sivakumar-ZS0528**
Issue: FINERACT-2035 
Client Trend For Day Scale is not work Due to BadSqlGrammar Error

This error is happened due to bucket 1 Day is inside single quotation  in 0087_update_dashboard_table_reports.xml file which is responsible for storing SQL query in DB that SQL query is responsible for the BadSQLGrammar Error

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by adamsaghy** [comment]
Please use the same formatting  in the commit message as it was used in the PR title.

"FINERACT-XXXX: description of the changes". You might wanna use the same as the PR title.

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0087_update_dashboard_table_reports.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
             FROM m_office o LEFT JOIN m_client cl on o.id = cl.office_id
                 LEFT JOIN m_loan ln on cl.id = ln.client_id
             WHERE o.hierarchy like concat((select ino.hierarchy from m_office ino where ino.id = ${officeId}),'%' )
-                AND (ln.disbursedon_date BETWEEN DATE_SUB(CURDATE(), INTERVAL 12 DAY) AND DATE(NOW()- INTERVAL '1 DAY'))
+                AND (ln.disbursedon_date BETWEEN DATE_SUB(CURDATE(), INTERVAL 12 DAY) AND DATE(NOW()- INTERVAL 1 DAY))
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0087_update_dashboard_table_reports.xml:78]: The same issue might appear in other reports as well. Can you please cover them in this PR?

Also please make sure you have an integration test which cover the issue was truly fixed

**Review by adamsaghy — CHANGES_REQUESTED** [review]
The same issue might appear in other reports as well. Can you please cover them in this PR?

Also please make sure you have an integration test which cover the issue was truly fixed

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0087_update_dashboard_table_reports.xml (hunk 1 above)
Comment (Mariappan-Sivakumar-ZS0528) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/0087_update_dashboard_table_reports.xml:78]: Ok sure, I'll review related reports to identify any instances of the same issue and incorporate the necessary fixes into the PR.

**Pull request closed without merging**